.travis.yml

```diff
@@ -4,15 +4,17 @@ env:
   - CODECLIMATE_REPO_TOKEN=ef69eb6a9e19987527e5010a02bbf71cd4ddc8a5137ba308f299817f56d06e7f
 script: "bundle exec rake spec"
 rvm:
-  - 2.0
```

Comment:
I know support has ended for 2.0 but why not put it in the `allow_failure` section as it is still quite fresh?